sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -839,6 +839,24 @@ object SQLConf {
     .checkValues(HiveCaseSensitiveInferenceMode.values.map(_.toString))
     .createWithDefault(HiveCaseSensitiveInferenceMode.NEVER_INFER.toString)
 
+  val HIVE_BLOBSTORE_SUPPORTED_SCHEMES =
+    buildConf("spark.sql.hive.blobstore.supported.schemes")
+      .doc("Comma-separated list of supported blobstore schemes.")
+      .version("3.1.0")
+      .stringConf
+      .transform(_.toLowerCase(Locale.ROOT))
+      .createWithDefault("s3,s3a,s3n")
+
+  val HIVE_BLOBSTORE_USE_BLOBSTORE_AS_SCRATCHDIR =
+    buildConf("spark.sql.hive.blobstore.use.blobstore.as.scratchdir")
+      .doc("Enable the use of scratch directories directly on blob storage systems " +
+        "(it may cause performance penalties). " +
+        "If you disable this parameter, Spark writes the data first in scratch dir, " +
+        "and move it to blobstore because moving it on blobstore is expensive.")
```

Comment:
Could you brush up the description more for this option? I think it is difficult for end users to understand when this option used inside Spark. For example, how about adding a statement, `... when  writing data out to a Hive table`?

Reply:
Modified as you suggested.